### PR TITLE
IfcSystemFurnitureElement Rail Import BIMQ 2021-07-21T07:33:18.1671031Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcSystemFurnitureElement

### CODE REVIEW

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SystemFurnitureElementTypeSubrack/DocPropertySet.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SystemFurnitureElementTypeSubrack" UniqueId="a1e902c5-5e10-bfe8-981c-d7c865152b2e" ApplicableType="IfcSystemFurnitureElement/SUBRACK" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NumberOfSlots_3ol50yqfz8ZAv5ApU0vpo5" />
+		<DocProperty xsi:nil="true" href="NumberOfUnits_21W3fiATv7tvnEAOCGtP5b" />
+		<DocProperty xsi:nil="true" href="NumberOfOccupiedUnits_1BOKOcsX1EP83zoqRMmf4n" />
+		<DocProperty xsi:nil="true" href="OccupancyRate_27M6QfrA5DOxxJQsKn0s4" />
+	</Properties>
+</DocPropertySet>
+
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SystemFurnitureElementTypeSubrack/DocPropertySet.xml:10]: Done. Commit : 26a84a52d1d2d5dd863865b663c2659ac68f2b7e

File: IFC4x3/Properties/o/OccupancyRate_27M6QfrA5DOx$xJQsKn0s4/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Indicates the ratio of occupied units to the total number of rack's vertical units. It is computed from the 2 other properties: NumberOfUnits and NumberOfOccupiedUnits.
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/o/OccupancyRate_27M6QfrA5DOx$xJQsKn0s4/Documentation.md]: I have erased OccupancyRate. Commit : bb6a5764b15b887e75d127b259399faa62eb2744

**Review by larswik — APPROVED** [review]

File: IFC4x3/Properties/n/NumberOfOccupiedUnits_1BOKOcsX1EP83zoqRMmf4n/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfOccupiedUnits_1BOKOcsX1EP83zoqRMmf4n" Name="NumberOfOccupiedUnits" UniqueId="4b614626-da10-4e64-80fd-cb46d6c29131" PrimaryDataType="IfcInteger" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfOccupiedUnits_1BOKOcsX1EP83zoqRMmf4n/DocProperty.xml:2]: Assuming this cannot be extracted from the model?

File: IFC4x3/Properties/n/NumberOfSlots_3ol50yqfz8ZAv5ApU0vpo5/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfSlots_3ol50yqfz8ZAv5ApU0vpo5" Name="NumberOfSlots" UniqueId="f2bc503c-d29f-488c-ae45-2b3780e73c85" PrimaryDataType="IfcInteger" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfSlots_3ol50yqfz8ZAv5ApU0vpo5/DocProperty.xml:2]: Same question as above.

File: IFC4x3/Properties/n/NumberOfSlots_3ol50yqfz8ZAv5ApU0vpo5/Documentation.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Indicate the number of slots of the device.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfSlots_3ol50yqfz8ZAv5ApU0vpo5/Documentation.md]:
```suggestion
Indicates the number of device slots.
```

File: IFC4x3/Properties/n/NumberOfUnits_21W3fiATv7tvnEAOCGtP5b/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Indicate the number of vertical units.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfUnits_21W3fiATv7tvnEAOCGtP5b/Documentation.md]:
```suggestion
Indicates the number of vertical units.
```

File: IFC4x3/Properties/n/NumberOfUnits_21W3fiATv7tvnEAOCGtP5b/DocProperty.xml [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfUnits_21W3fiATv7tvnEAOCGtP5b" Name="NumberOfUnits" UniqueId="81803a6c-29de-47df-9c4e-298310dd9165" PrimaryDataType="IfcInteger" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfUnits_21W3fiATv7tvnEAOCGtP5b/DocProperty.xml:2]: Same question as above.

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Some changes requested.

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**